ci/image/build.sh

```diff
@@ -14,10 +14,10 @@ for name in "${!BUILD_IMAGE_NAMES[@]}"; do
         --tag ${BUILD_IMAGE_NAMES[$name]} .
 done
 
-for name in "${!TEST_IMAGE_NAMES[@]}"; do
-    docker build \
-        --file $THIS_DIR/Dockerfile.$name-test \
-        --label snowflake \
-        --label $DRIVER_NAME \
-        --tag ${TEST_IMAGE_NAMES[$name]} .
-done
+#for name in "${!TEST_IMAGE_NAMES[@]}"; do
```

Comment:
Same comment as above.

Reply:
It's the same block but commented. They are commented out since the centos images are not being run anymore due to being incompatible with the later versions of node.